cmd/system-probe/subcommands/run/command.go

```diff
@@ -65,7 +66,7 @@ func Commands(globalParams *command.GlobalParams) []*cobra.Command {
 				fx.Supply(cliParams),
 				fx.Supply(config.NewAgentParamsWithoutSecrets("", config.WithConfigMissingOK(true))),
 				fx.Supply(sysprobeconfig.NewParams(sysprobeconfig.WithSysProbeConfFilePath(globalParams.ConfFilePath), sysprobeconfig.WithConfigLoadSecrets(true))),
-				fx.Supply(log.LogForDaemon("SYS-PROBE", "log_file", common.DefaultLogFile)),
```

Comment:
This `common` is `github.com/DataDog/datadog-agent/cmd/system-probe/common` not `github.com/DataDog/datadog-agent/cmd/agent/common`

Reply:
If I wasn't clear, this needs to change. Currently is it changing the default log file from `/var/log/datadog/system-probe.log` to `/var/log/datadog/agent.log`